### ISSUE TITLE
[SRSim]: HTML - image with image maps

### ISSUE THREAD

**Issue by tombrunet** [IBMa/equal-access#2360]
When an image map is used, NVDA is not reading the alt text of the image. We are. Investigate this further - feels like the alt text should be read (we are correct?)

**Comment by MHoov** [comment]
moving to R7.3
